internal/worker_runner.go

```diff
@@ -124,6 +140,32 @@ func (wr *workerRunner) Run() {
 	}
 }
 
+// DeviceStatusData is formatted as json, gzip compressed, then base64 compressed.
+// This is done to reduce the size of the payload sent to the cloud over MQTT.
+func compressDeviceStatusData(s models.DeviceStatusData) (*models.DeviceStatusCompressedData, error) {
+	payload, err := json.Marshal(s)
+	if err != nil {
+		return nil, err
+	}
+	println(string(payload))
```

Comment:
do we still need this println or was that for debug?

Reply:
yep, it was just for debugging purposes